desc/objectives/_equilibrium.py

```diff
@@ -159,6 +159,14 @@ def compute(self, *args, **kwargs):
             Spectral coefficients of I(rho) -- toroidal current profile.
         Psi : float
             Total toroidal magnetic flux within the last closed flux surface (Wb).
+        Te_l : ndarray
+            Spectral coefficients of Te(rho) -- electron temperature profile.
+        ne_l : ndarray
+            Spectral coefficients of ne(rho) -- electron density profile.
+        Ti_l : ndarray
+            Spectral coefficients of Ti(rho) -- ion temperature profile.
```

Comment:
We should include units for these parameters -- `(eV)` and `(1/m^3)` -- like we do for all the other parameters that are not dimensionless. Same for all other objective function documentations.